rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/util/PackerManifestController.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2018 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.rosco.providers.util;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.netflix.spinnaker.rosco.providers.util.PackerManifest.PackerBuild;
+import java.io.BufferedInputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+
+public class PackerManifestController {
```

Comment:
Usually we reserve the word "controller" for web endpoints, e.g. spring `@RestController`s. This might best be called a "service"?